bayesian/api_v1.py

```diff
@@ -808,18 +830,22 @@ def get(self, external_request_id):
                             if recommendation.get("input_stack", {}).get("appstack_id", "") != "":
                                 uri = "{analytics_baseurl}/api/v1.0/appstack/{appstack_id}"\
                                       .format(analytics_baseurl=url,
-                                              appstack_id=recommendation["input_stack"]["appstack_id"])
+                                              appstack_id=recommendation["input_stack"]
+                                              ["appstack_id"])
                                 recommendation["input_stack"]["uri"] = uri
 
-                            if recommendation.get("recommendations", {}).get("similar_stacks", "") != "":
+                            if recommendation.get("recommendations", {}).\
+                               get("similar_stacks", "") != "":
                                 for r in recommendation["recommendations"]["similar_stacks"]:
```

Comment:
Are you going to fix this `if` and `for` (as we've discussed) ?

Reply:
@jpopelka I'm going to...in separate commit/PR